src/components/views/dashboard.tsx

```diff
@@ -76,6 +77,31 @@ export const Dashboard: FC<any> = ({navigation}) => {
     />
   );
 
+  const type = exposure.status.type ? exposure.status.type : [];
```

Comment:
Small style thing: this is a big file and `type` is a very generic name, and only used once inside `renderAlertInfo`. Can it be moved inside `renderAlertInfo` (and also changed to `= exposure.status.type || [];`)